rkt/api_service_stub.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build !sdjournal
+// +build !sdjournal,linux
```

Comment:
I think this is a redundant combination, as I guess on Mac you need to build with `--enable-sdjournal=no` anyway. Am I correct?